### DISCUSSION TITLE
Executable file in different computer shows as virus

### DISCUSSION THREAD

**Discussion by khanalim** [brentvollebregt/auto-py-to-exe#334] [Q&A] [answered]
I use Mac and I created an exe file from python script using PyInstaller. Exe file works in my pc very well. However, whenever, I take this file to other computer regardless of Mac or windows, exe file does not work. The file is considered as security risk. The other pc does not even recognize this exe file as exe file. Any suggestions to solve this issue?

**Comment by brentvollebregt** [comment, marked as answer]
If the executable was built on Windows, it will only run on Windows.

In terms of the "security risk" - this could be potentially one of these things (I'm not sure what you're seeing so have to take a guess):
- Windows defender saying the exe is a virus - take a look at "[My Antivirus Detected the exe as a Virus](https://nitratine.net/blog/post/issues-when-using-auto-py-to-exe/#my-antivirus-detected-the-exe-as-a-virus)"
- Windows defender warning about no publisher - check out this [SO answer](https://stackoverflow.com/a/45316660)

**Reply by khanalim** [reply]
Thank you for answering my question. I really appreciate.
